package.json

```diff
@@ -33,6 +33,8 @@
       "build/auth/pages/*.ejs",
       "build/hooks",
       "node_modules/resin-discoverable-services/services/**/*",
+      "node_modules/opn-cli/node_modules/opn/xdg-open",
+      "node_modules/open/xdg-open",
```

Comment:
I believe the 2 lines above are redundant (`pkg` would automatically include the `xdg-open` script in the standalone executable anyway), however I thought it would be desirable to make the dependency **explicit.** Also, in case dependencies are updated such that `node_modules/opn-cli/node_modules/opn/xdg-open` "disappears", we would quickly find out about it and this would prompt us to consider what else needs to be changed -- like the patch files in the `patches` folder.